### DISCUSSION TITLE
各位大佬， 怎么设置 passwall或passwall2， 可以防止DNS/WebRTC  泄露啊。。求教！！！可有偿！ 访问控制，多个节点都要防泄露。。。

### DISCUSSION THREAD

**Discussion by Zren215** [Openwrt-Passwall/openwrt-passwall2#683] [Q&A]
各位大佬， 怎么设置 passwall或passwall2， 可以防止DNS/WebRTC  泄露啊。。求教！！！可有偿！
访问控制，多个节点都要防泄露。。。

**Comment by HSK077** [comment]
https://burgess-t.cn:35006/index.php/2024/07/30/617.html

**Comment by Kristinwsio** [comment]
webrtc泄漏至今还没有搜到能解决的
